plugin/file/secondary.go

```diff
@@ -112,6 +114,8 @@ func (z *Zone) Update() error {
 	}
 	retryActive := false
 
+	server := metrics.WithServer(context.Background())
```

Comment:
this can't work, and can't give you the server. Which is a problem, because I don't think we have another way to get to it...

Reply:
So should I drop the server label from these metrics? because I tried finding other ways for it in the codebase but couldnt